CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ option(EQUALIZER_USE_HWLOC "Test for hwloc support (experimental)" OFF)
 option(EQUALIZER_INSTALL_SERVER_HEADERS "Install Equalizer server headers" OFF)
 option(EQUALIZER_BUILD_2_0_API
   "Enable for pure 2.0 API (breaks compatibility with 1.x API)" OFF)
+option(EQUALIZER_PREFER_QT "Prefer Qt window system over X11 on OS X" OFF)
```

Comment:
It's not as complete as X11 (no fonts). I don't have a strong opinion, can switch it to ON if you prefer.